File: Taller/polimorfismo/Triangulo.php
<?php
require_once './FiguraGeometrica.php';
class Triangulo extends FiguraGeometrica
{
  protected $base;
  protected $altura;

  public function __construct($color, $base, $altura)
  {
    parent::__construct($color, $base, $altura);
  }
}

File: JavaProject/src/main/java/org/unimagdalena/geometria/Triangulo.java
package org.unimagdalena.geometria;

import lombok.Builder;
import lombok.Data;
import org.unimagdalena.interfaces.FiguraGeometrica;

@Builder
@Data
public class Triangulo implements FiguraGeometrica {

    private double base;
    private double altura;
    private double areaTriangulo;

    @Override
    public double calcularArea() {
       return areaTriangulo = (base * altura)/2;
    }
}
